src/main/fc/cli.c

```diff
@@ -3714,6 +3714,17 @@ static void cliSet(char *cmdline)
                 }
 
                 if (changeValue) {
+                    // If changing the battery capacity unit, update the osd stats energy unit to match
+                    if (strcmp(name, "battery_capacity_unit") == 0) {
```

Comment:
Any reason to not just merge osdConfig()->stats_energy_unit and battery_capacity_unit?

Reply:
Some people may still want to see mAh/distance in stats. For comparing with friends. Some crazy people don't use Wh ;)